Tests/kaas/k8s-node-distribution/k8s-node-distribution-check.py

```diff
@@ -225,7 +212,13 @@ async def main(argv):
         "topology.scs.community/host-id",
     )
 
-    nodes = await get_k8s_cluster_labelled_nodes(config.kubeconfig, labels + ("node-role.kubernetes.io/control-plane", ))
+    if isinstance(config.testconfig, dict):
+        nodes = [v for _, v in config.testconfig.items()]
+    else:
+        nodes = await get_k8s_cluster_labelled_nodes(
+            config.kubeconfig,
+            labels + ("node-role.kubernetes.io/control-plane", )
+        )
```

Comment:
Instead of this, I would suggest to split the `main` function and have another function, e.g., `check_nodes`, that takes a `nodes` a parameter. This function can then easily be unit tested with static data instead of the dynamic "real" data from `get_k8s_cluster_labelled_nodes`.

Reply:
Did some changes here https://github.com/SovereignCloudStack/standards/pull/558/commits/aec75ef610ce03dc848181b9a85fefc6354b6ed0